MAUI/ImageEditor/customview.md

```diff
@@ -210,7 +210,7 @@ Add a custom view on image loading using the [`Imageloaded`](https://help.syncfu
 {% endtabs %}
 
 
-### Add custom view with manual bounds
+## Add custom view with manual bounds
```

Comment:
Do these changes for text and shape files

Reply:
Updated